docs/source/enable/mouse_events.rst

```diff
@@ -0,0 +1,52 @@
+Enable Mouse Events
+===================
+
+Enable mouse events are represented by the :class:`MouseEvent` type and their
```

Comment:
This doesn't link to the api docs but we may want it to

Reply:
I'm not sure how to do that.

Reply:
Figured it out.

Reply:
I believe you can replace ":class:`MouseEvent`" with ":class:`~enable.events.MouseEvent`"

Reply:
":class:`~.MouseEvent`" also works if there's only one `MouseEvent` (which there is).